templates/sendemailaction/email/message.txt.vm

```diff
@@ -0,0 +1,48 @@
+#*
```

Comment:
wrong copyright header

Reply:
Good catch, thanks! It's now fixed in both.